containers/proxy-tftpd-image/tftp_wrapper.py

```diff
@@ -295,6 +345,7 @@ def main():
         with open(args.configFile) as source:
             config = yaml.safe_load(source)
             args.proxyFqdn = config["proxy_fqdn"]
+            args.serverFqdn = config["server"]
```

Comment:
?
```suggestion
            args.serverFqdn = config["server_fqdn"]
```

In the same style as proxy above.

Reply:
Can't do that unless we want to make existing configs unusable to users and also change how we generate proxy config in the java.